### ISSUE TITLE
Cinnamon Menu Applet Modernization 

### ISSUE THREAD

**Issue by agentl074** [linuxmint/cinnamon#11334]
One feature that I believe would be appreciated would be a way to customize the start menu applet to look like the centralized Windows 11 or Gnome apps theme as an option. Thank you!

**Comment by Secret-chest** [comment]
This is not how it works. Just use another applet. And if you want to centre it, right-click the panel, enable panel edit mode, drag the menu and window list to the centre, then disable panel edit mode.

**Comment by agentl074** [comment]
Yeah, I think I will just stick with Ubuntu for that more modern look. I was just hoping that Cinnamon would be more modernized in a future release, but I guess that's why there's Gnome.

**Comment by Secret-chest** [comment]
> Yeah, I think I will just stick with Ubuntu for that more modern look. I was just hoping that Cinnamon would be more modernized in a future release.... Update... I have found a more modern menu called Cinnamenu -- this looks like it is what I am seeking. 

That's what I said... just use another menu, you don't need the stock one
